### PR TITLE
bump to 1.19

### PR DESCRIPTION
**Pull request by polylymer**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1 +1,2 @@
-kotlin.code.style=official
\ No newline at end of file
+kotlin.code.style=official
+org.gradle.jvmargs=-Xmx3G
```
Comment (mooziii) [inline gradle.properties:2]: why 3gb

File: build.gradle.kts [hunk 2]
```diff
@@ -23,18 +23,18 @@ repositories {
...
     // event system
     implementation("me.obsilabor:alert:1.0.5")
     include("me.obsilabor:alert:1.0.5")
     // fabric
-    minecraft("com.mojang:minecraft:1.19-pre1")
-    mappings("net.fabricmc:yarn:1.19-pre1+build.1")
-    modImplementation("net.fabricmc:fabric-loader:0.14.5")
+    minecraft("com.mojang:minecraft:1.19")
+    mappings("net.fabricmc:yarn:1.19+build.1")
+    modImplementation("net.fabricmc:fabric-loader:0.14.6")
```
Comment (mooziii) [inline build.gradle.kts:33]: This just makes this mod more incompatible with older loader versions

**Review by mooziii — CHANGES_REQUESTED** [review]

File: gradle.properties (hunk 1 above)
Comment (mooziii) [reply gradle.properties:2]: 1gb works for me

File: build.gradle.kts (hunk 2 above)
Comment (polylymer) [reply build.gradle.kts:33]: People who use an old fabric loader have only themselves to blame 🙃

Comment (polylymer) [reply build.gradle.kts:33]: btw i will not retract the changes, either you accept the changes or close the pull request 🤔

**Pull request merged**